share/locale/en.json

```diff
@@ -1,165 +1,165 @@
+    "settings_test_success": "Connected to the Tor controller.\n\nTor version: {}\nSupports ephemeral onion services: {}.\nSupports onion services that are stealthy: {}.",
+    "error_tor_protocol_error": "Could not communicate with the Tor controller.\nIf you are using Whonix, check out https://www.whonix.org/wiki/onionshare to make OnionShare work.",
     "connecting_to_tor": "Connecting to the Tor network",
-    "update_available": "A new version of OnionShare is available. <a href='{}'>Click here</a> to download it.<br><br>Installed version: {}<br>Latest version: {}",
-    "update_error_check_error": "Error checking for updates: Maybe you're not connected to Tor, or maybe the OnionShare website is down.",
-    "update_error_invalid_latest_version": "Error checking for updates: The OnionShare website responded saying the latest version is '{}', but that doesn't appear to be a valid version string.",
-    "update_not_available": "You are running the latest version of OnionShare.",
-    "gui_tor_connection_ask": "Would you like to open OnionShare settings to troubleshoot connecting to Tor?",
-    "gui_tor_connection_ask_open_settings": "Open Settings",
+    "update_available": "New version of OnionShare. <a href='{}'>Click here</a> to get it.<br><br>You have {} and the latest is {}.",
+    "update_error_check_error": "Could not check for new versions: Maybe you're not connected to Tor, or the OnionShare website is down?",
+    "update_error_invalid_latest_version": "Could not check for new versions: The OnionShare website is saying the latest version is the unrecognizable '{}'…",
+    "update_not_available": "You are running the latest OnionShare.",
+    "gui_tor_connection_ask": "Would you like to open the settings to troubleshoot connecting to Tor?",
+    "gui_tor_connection_ask_open_settings": "Yes",
     "gui_tor_connection_ask_quit": "Quit",
-    "gui_tor_connection_error_settings": "Try adjusting how OnionShare connects to the Tor network in Settings.",
-    "gui_tor_connection_canceled": "OnionShare could not connect to Tor.\n\nMake sure you're connected to the Internet, then re-open OnionShare to set up the Tor connection.",
+    "gui_tor_connection_error_settings": "Try changing how OnionShare connects to the Tor network in the settings.",
+    "gui_tor_connection_canceled": "Could not connect to Tor.\n\nEnsure you are connected to the Internet, then re-open OnionShare and set up its connection to Tor.",
     "gui_tor_connection_lost": "Disconnected from Tor.",
-    "gui_server_started_after_timeout": "The server started after your chosen auto-timeout.\nPlease start a new share.",
-    "gui_server_timeout_expired": "The chosen timeout has already expired.\nPlease update the timeout and then you may start sharing.",
-    "share_via_onionshare": "Share via OnionShare",
+    "gui_server_started_after_timeout": "The expiry timer ran out before the server started.\nPlease start a new share.",
```

Comment:
Maybe just: "The auto-stop timer ran out before the server started.\nPlease start a new share."

Reply:
"The set timeframe for OnionSharing ran out before the server started.\nPlease make a new OnionShare".=